.github/workflows/main.yml

```
@@ -129,18 +130,27 @@ jobs:
           yapf -d $CHANGED_FILES
         fi
     - name: Flake8
+      id: flake8
```

Maybe with something like this?

```suggestion
      id: flake8
      continue-on-error: true
```

---

I saw that option in the documentation, but the description is unclear.  If we use it, will that cause it to report that the run succeeded even if that step failed?